agency/README.md

```diff
@@ -222,6 +175,23 @@ List of valid vehicle events and the resulting vehicle status if the event is su
 | `maintenance_pick_up`  | Vehicle removed from street so it can be worked on                                                   | `available`, `unavailable`                         | `removed`           |                                                                         |
 | `deregister`           | A vehicle is deregistered                                                                            | `available`, `unavailable`, `removed`, `elsewhere` | `inactive`          | A vehicle is deactivated from the fleet and unavailable.                |
 
+## Telemetry Data
+
+A standard point of vehicle telemetry. References to latitude and longitude imply coordinates encoded in the [WGS 84 (EPSG:4326)](https://en.wikipedia.org/wiki/World_Geodetic_System) standard GPS projection expressed as [Decimal Degrees](https://en.wikipedia.org/wiki/Decimal_degrees).
+
+| Field          | Type           | Required/Optional     | Field Description                                            |
+| -------------- | -------------- | --------------------- | ------------------------------------------------------------ |
+| `vehicle_id`   | UUIDv4         | Required              | ID used in [Register](#vehicle-register)                     |
+| `timestamp`    | Unix Timestamp | Required              | Date/time that event occurred. Based on GPS clock            |
```

Comment:
This section https://github.com/CityOfLosAngeles/mobility-data-specification/tree/dev/provider#timestamps could also be added to this spec to clarify what timestamp is.

Reply:
Will add to future PR to align with provider spec in general.